### PR TITLE
2 new Sinful Demon powers

### PR DESCRIPTION
**Pull request by Marmio64**
Not very big or especially interesting powers, but they are in dire need of something.

Wrath and Envy get the ability Torment
Empowers your hand to allow you to strike someone to inflict a wide variety of debilitating effects. Blurs vision, inflicts hallucinations and disgust, causes minor brain damage, very short immobilization and stun, and confusion. Most useful for torture as Wrath or to inflict and then run for either. 20 second cooldown

Pride gets the Mend ability
Also empowers your hand, but instead with healing powers to recover 20 brute and 20 burn on a target (but not yourself). Pride is meant to be pseudo helpful to the crew (and sometimes other antagonists) so this fits in with them helping others. 12 second cooldown

also a few bug fixes that i noticed how to fix while working on this

# Changelog

:cl:  
rscadd: New Sinful demon abilities. Torment for Wrath and Envy. Mend for Pride.  
bugfix: Demon form works again. Touch attacks work properly. Icon borders fixed, and ignite works again.
/:cl:

### CODE REVIEW

**Comment by Marmio64** [comment]
I'll see about fixing conflicts later tonight or tomorrow.

File: code/modules/antagonists/demon/general_powers.dm [hunk 1]
```diff
@@ -72,3 +74,49 @@
...
+	
+
+/obj/item/melee/touch_attack/torment
+	name = "Vindictive Hand"
+	desc = "An utterly scornful mass of hateful energy, ready to strike."
+	icon_state = "flagellation"
+	item_state = "hivemind"
+
+/datum/action/cooldown/spell/touch/torment/cast_on_hand_hit(obj/item/melee/touch_attack/hand, atom/victim, mob/living/carbon/caster)
+	var/mob/living/M = victim
```
Comment (JohnFulpWillard) [inline code/modules/antagonists/demon/general_powers.dm]: Why not just path ``victim`` to ``/mob/living`` in args then?

File: code/modules/antagonists/demon/sins/pride.dm [hunk 2]
```diff
@@ -14,3 +15,39 @@
...
+	hand_path = /obj/item/melee/touch_attack/mend
+
+/obj/item/melee/touch_attack/mend
+	name = "Mending Hand"
+	desc = "A seemingly pleasant mass of mending energy, ready to heal."
+	icon_state = "flagellation"
+	item_state = "hivemind"
+
+/datum/action/cooldown/spell/touch/mend/cast_on_hand_hit(obj/item/melee/touch_attack/hand, atom/victim, mob/living/carbon/caster)
+	var/mob/living/M = victim
```
Comment (JohnFulpWillard) [inline code/modules/antagonists/demon/sins/pride.dm]: Ditto

File: code/modules/antagonists/demon/demons.dm [hunk 3]
```diff
@@ -29,7 +29,9 @@
 		/datum/action/cooldown/spell/conjure/summon_greedslots,
 		/datum/action/cooldown/spell/pointed/ignite,
 		/datum/action/cooldown/spell/touch/envy,
-		/datum/action/cooldown/spell/conjure/summon_mirror
+		/datum/action/cooldown/spell/conjure/summon_mirror,
+		/datum/action/cooldown/spell/touch/mend,
+		/datum/action/cooldown/spell/touch/torment
```
Comment (JohnFulpWillard) [inline code/modules/antagonists/demon/demons.dm]:
```suggestion
		/datum/action/cooldown/spell/touch/torment,
```

File: code/modules/antagonists/demon/general_powers.dm [hunk 4]
```diff
@@ -74,3 +74,48 @@
...
+		..()
+		return TRUE
+	playsound(caster, 'sound/magic/demon_attack1.ogg', 75, TRUE)
+	victim.blur_eyes(15) //huge array of relatively minor effects.
+	victim.adjust_jitter(5 SECONDS)
+	victim.set_confusion_if_lower(5 SECONDS)
+	victim.adjust_disgust(40)
+	victim.adjust_hallucinations(20 SECONDS)
+	victim.Immobilize(30)
+	victim.Stun(10)
```
Comment (Moltijoe) [inline code/modules/antagonists/demon/general_powers.dm]:
```suggestion
	victim.Immobilize(3 SECONDS)
	victim.Stun(1 SECONDS)
```

**Review by Moltijoe — CHANGES_REQUESTED** [review]

**Review by Moltijoe — APPROVED** [review]
looks to be fine

**Pull request merged**